script/1_0_2current_en_script/Teach_Me_Ciel-Sensei/QA_04.txt

```diff
@@ -43,7 +43,7 @@ Oh really? That was unexpected.%{n}
 You don't like your steak a little bloody even though%{n}you're from France?
 <Page16>
 I like tender meat the most when it comes to beef curry,
-but rare steak is a bit... much.//this is not 'mentally', quite the opposite. 生理的 literally means 'physiologically'. Think of it more like 'instinctively' or something like that. Also, with how Japanese works with implications and what not, it might be worth adding 'too much' after 'a bit' here. In English thoughts aren't ended halfway like this.
+ but rare steak is a bit... much.//this is not 'mentally', quite the opposite. 生理的 literally means 'physiologically'. Think of it more like 'instinctively' or something like that. Also, with how Japanese works with implications and what not, it might be worth adding 'too much' after 'a bit' here. In English thoughts aren't ended halfway like this.
```

Comment:
Again, why space added at the start?
> But I mean for your health, rare steak is a bit...

Reply:
Hacky line break things :) Please never remove extra spaces at the beginning of lines in QA files. They are to prevent run-ons which in turn prevent text box overflow.